cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsDiscoveryHandler.java

```diff
@@ -55,14 +60,15 @@
 public final class MetricsDiscoveryHandler extends BaseMetricsHandler {
```

Comment:
please create JIRA to cleanup it after we remove 2.0 UI support and add todo linking to it here.